classes/passwd.oeclass

```diff
@@ -19,18 +19,14 @@ GROUP_FILES ?= "${SRCDIR}/group"
 PASSWD_POSTFUNC:machine = " do_install_passwd"
 do_install[postfuncs] += "${PASSWD_POSTFUNC}"
 do_install_passwd () {
-	i=0
 	for f in ${PASSWD_FILES} ; do
-		mkdir -p ${D}/${passwddir}
-		let i=$i+1
-		cp $f ${D}/${passwddir}/${PN}.$i
+		mkdir -p ${D}/${passwddir}/${PN}
+		cp $f ${D}/${passwddir}/${PN}/
```

Comment:
Agreed; (2) also occured to me after I posted this. But I'd still like to find a way to name and place the passwd files under ${passwddir} in a completely predictable manner so it's easier to write correct FILES variables. In particular, it should be possible to do PASSWD_FILES += "foo-passwd", FILES_${PN}-foo += "some_function_of(foo-passwd)" anywhere in a recipe without having to worry about what other entities PASSWD_FILES contained at that point.